### PR TITLE
limit audio senders, as was done for video senders.

### PR DESCRIPTION
**Pull request by jqdrqgnq**

### CODE REVIEW

File: src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java [hunk 1]
```diff
@@ -296,13 +304,26 @@ void processPresence(Presence presence)
             statsId = statsIdPacketExt.getStatsId();
         }
 
+        boolean wasAudioMuted = isAudioMuted;
+        StandardExtensionElement audioMutedExt = presence.getExtension("audiomuted", "jabber:client");
```
Comment (bgrozev) [inline src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java:308]: Ingo has been doing some work around extensions in #832 and he fixed `VideoMutedExtension`. Can you please use the same approach for audio?

File: src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java [hunk 2]
```diff
@@ -296,13 +304,26 @@ void processPresence(Presence presence)
...
+        isAudioMuted = audioMutedExt == null || Boolean.valueOf(audioMutedExt.getText()); /* defaults to true */
+
+        if (isAudioMuted != wasAudioMuted)
+        {
+            logger.debug(() -> toString() + ". isAudioMuted = " + isAudioMuted + ".");
```
Comment (bgrozev) [inline src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java:313]: If you need extra context for the `ChatMemberImpl` instance in the logs you can add it to the logger, e.g.
`logger.addContext("occupantJid", occupantJid.toString());`. We could also hook it up to the "parent" logger so it has context for the chat room/conference.

File: src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java (hunk 1 above)
Comment (jqdrqgnq) [reply src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java]: done

File: src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java (hunk 2 above)
Comment (jqdrqgnq) [reply src/main/java/org/jitsi/impl/protocol/xmpp/ChatMemberImpl.java]: added occupantJid to the logging context.

**Comment by jqdrqgnq** [comment]
Requires #839 to eliminate the accumulating of old conference-properties elements within presence updates.

**Review by bgrozev — APPROVED** [review]

**Pull request merged**